### ISSUE TITLE
Public key

### ISSUE THREAD

**Issue by kurt-by** [WeAreFairphone/android_device_fairphone_FP3#29]
Could you please sign the build with a private key?

**Comment by mstaz** [comment]
Yes, I can. I tried it. Unfotunately it can't be easily updated with LOS recovery if the key changes. Must be done with TWRP then. Means only I can build images which can be used for easy update. If that is fine for all, I can definitely do that.

**Comment by irgendwer-2020** [comment]
Yes, doing this would be nice. Please do it for us.
